packages/dev/src/index.tsx

```diff
@@ -21,6 +21,8 @@ import AppLocalized from './AppLocalized';
 import { DefaultCatalog } from './DefaultCatalog';
 import { CustomCatalog } from './CustomCatalog';
 
+import './patternfly-theme-dark.css'; // @patternfly/patternfly/patternfly-theme-dark.css after PF dark theme release
```

Comment:
Since PF dark theme is already released. We'd better bump the version of patternfly package and use the CSS file inside the package and remove the manual imported one. Then test all the styles again.